doc/releasenotes/14_0_0_summary.md

```diff
@@ -187,3 +187,28 @@ The throttler now checks in with the heartbeat writer to request heartbeats, any
 When `--heartbeat_on_demand_duration` is not set, there is no change in behavior.
 
 When `--heartbeat_on_demand_duration` is set to a positive value, then the throttler ensures that the heartbeat writer generated heartbeats for at least the following duration. This also means at the first throttler check, it's possible that heartbeats are idle, and so the first check will fail. As heartbeats start running, followup checks will get a more accurate lag evaluation and will respond accordingly. In a sense, it's a "cold engine" scenario, where the engine takes time to start up, and then runs smoothly.
+
+### Durability Policy
+
+#### Deprecation of durability_policy Flag
+The flag `durability_policy` added to vtctl, vtctld and vtworker binaries has now been deprecated.
```

Comment:
this wording is confusing to me; it _sounds_ like we just added the flag to vtctl, vtctld, and vtworker, and are also deprecating it.

suggestion:

> The durability policy for a keyspace is now stored in the keyspace record in the topo server.
> The `durability_policy` flag used by `vtctl`, `vtctld`, and `vtworker` binaries has been deprecated.

Reply:
Fixed